assets/src/components/vehicleIcon.tsx

```diff
@@ -21,7 +21,7 @@ export interface Props {
   label?: string
   variant?: string | null
   status?: DrawnStatus
-  alertIcon?: boolean
+  alertIcon?: AlertIconStyle
```

Comment:
To me, this sounds like you are passing in an icon. Would `alertIconStyle` be more descriptive?